2023-05-25 20:14:12,577 INFO    MainThread:272531 [wandb_setup.py:_flush():68] Configure stats pid to 272531
2023-05-25 20:14:12,578 INFO    MainThread:272531 [wandb_setup.py:_flush():68] Loading settings from /home/jjia/.config/wandb/settings
2023-05-25 20:14:12,578 INFO    MainThread:272531 [wandb_setup.py:_flush():68] Loading settings from /data1/jjia/lung_function/lung_function/scripts/jupyter_notebooks/wandb/settings
2023-05-25 20:14:12,579 INFO    MainThread:272531 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-05-25 20:14:12,579 INFO    MainThread:272531 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-05-25 20:14:12,580 INFO    MainThread:272531 [wandb_init.py:_log_setup():478] Logging user logs to /data1/jjia/lung_function/lung_function/scripts/jupyter_notebooks/wandb/run-20230525_201412-k382vss0/logs/debug.log
2023-05-25 20:14:12,581 INFO    MainThread:272531 [wandb_init.py:_log_setup():479] Logging internal logs to /data1/jjia/lung_function/lung_function/scripts/jupyter_notebooks/wandb/run-20230525_201412-k382vss0/logs/debug-internal.log
2023-05-25 20:14:12,581 INFO    MainThread:272531 [wandb_init.py:_jupyter_setup():428] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x2aaadbda4940>
2023-05-25 20:14:12,582 INFO    MainThread:272531 [wandb_init.py:init():518] calling init triggers
2023-05-25 20:14:12,582 INFO    MainThread:272531 [wandb_init.py:init():521] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.01, 'epochs': 10}
2023-05-25 20:14:12,583 INFO    MainThread:272531 [wandb_init.py:init():571] starting backend
2023-05-25 20:14:12,583 INFO    MainThread:272531 [wandb_init.py:init():575] setting up manager
2023-05-25 20:14:12,591 INFO    MainThread:272531 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-05-25 20:14:12,602 INFO    MainThread:272531 [wandb_init.py:init():582] backend started and connected
2023-05-25 20:14:12,630 INFO    MainThread:272531 [wandb_run.py:_label_probe_notebook():1202] probe notebook
2023-05-25 20:14:12,634 INFO    MainThread:272531 [wandb_run.py:_label_probe_notebook():1212] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-05-25 20:14:12,635 INFO    MainThread:272531 [wandb_init.py:init():670] updated telemetry
2023-05-25 20:14:12,738 INFO    MainThread:272531 [wandb_init.py:init():710] communicating run to backend with 60.0 second timeout
2023-05-25 20:14:13,158 INFO    MainThread:272531 [wandb_run.py:_on_init():2121] communicating current version
2023-05-25 20:14:13,229 INFO    MainThread:272531 [wandb_run.py:_on_init():2125] got version response upgrade_message: "wandb version 0.15.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-05-25 20:14:13,229 INFO    MainThread:272531 [wandb_init.py:init():758] starting run threads in backend
2023-05-25 20:14:19,495 WARNING MsgRouterThr:272531 [router.py:message_loop():77] message_loop has been closed
2023-05-25 20:14:28,270 ERROR   MainThread:272531 [wandb_init.py:init():1135] error
Traceback (most recent call last):
  File "/home/jjia/.conda/envs/py38/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1108, in init
    run = wi.init()
  File "/home/jjia/.conda/envs/py38/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 770, in init
    run_start_result = run_start_handle.wait(timeout=30)
  File "/home/jjia/.conda/envs/py38/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 271, in wait
    raise MailboxError("transport failed")
wandb.errors.MailboxError: transport failed
